# Request a plan to a random path
int64 n_waypoints
---
bool success
